apps/designer/app/shared/router-utils/path-utils.ts

```diff
@@ -68,11 +68,17 @@ export const getBuildUrl = ({
   project,
   page,
   mode,
+  // Hidden token for canvas (should not be visible in browser URL)
+  readToken,
```

Comment:
Is showing the token in some url a security risk?

Reply:
yes, thats why I gave it different name